lib/PluginHandler.js

```diff
@@ -114,9 +107,9 @@ class PluginHandler {
     /**
      * Initialize one Plugins
      *
-     * @param name {string} name of the plugin
-     * @param parentConfig {object} io-package of the parent module that uses the plugins (adapter/controller)
-     * @param callback {function} callback function which is called after initialization is done for all plugins
+     * @param {string} name name of the plugin
+     * @param {Record<string, any>} parentConfig io-package of the parent module that uses the plugins (adapter/controller)
+     * @param {(error?: string) => void} callback callback function which is called after initialization is done for all plugins
```

Comment:
Function is too unspecific. This way, consumers get checks for the method signature.